ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -218,8 +269,24 @@ open class AvatarView: UIView {
         imageView.frame = bounds
         initialsView.frame = imageView.frame
 
+        let presenceSize = avatarSize.presenceSize.sizeValue
+        let presenceCornerOffset = style == .circle ? avatarSize.presenceCornerOffset : 0.0
+        var presenceFrame = CGRect(x: bounds.width - presenceSize - presenceCornerOffset,
```

Comment:
RTL works fine?

Reply:
I need to ask the designers if we want to flip this UI in RTL. I assumed that we don't.

Reply:
i heard rumors Teams.app do flip but not sure it does. ...

Reply:
The Teams app does flip the presence status so let's follow that design. I'll update the logic here to support RTL.